src/providers/tfc/workspaceProvider.ts

```diff
@@ -112,10 +113,25 @@ export class WorkspaceTreeDataProvider implements vscode.TreeDataProvider<Worksp
         const workspace = workspaces[index];
 
         const project = projects.find((p) => p.id === workspace.relationships.project.data.id);
-
         const projectName = project ? project.attributes.name : '';
 
-        items.push(new WorkspaceTreeItem(workspace.attributes.name, workspace.id, projectName, organization));
+        const lastRunId = workspace.relationships['latest-run']?.data?.id;
+        const lastestRun = workspaceResponse.included
+          ? workspaceResponse.included.find((run) => run.id === lastRunId)
+          : undefined;
+        const link = vscode.Uri.joinPath(vscode.Uri.parse(this.baseUrl), workspace.links['self-html']);
+
+        items.push(
+          new WorkspaceTreeItem(
+            workspace.attributes.name,
+            workspace.id,
+            projectName,
+            organization,
+            workspace,
+            link,
+            lastestRun?.attributes,
```

Comment:
```suggestion
            workspace.id,
            projectName,
            organization,
            workspace.attributes,
            link,
            lastestRun?.attributes,
```
Nitpick: It would be nice to slightly reduce the number of arguments and avoid passing through things we don't actually need.

Reply:
Which one isn't required here?